### PR TITLE
docs(list): update usage examples.

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** N/A

## Summary

- Update list usage examples to include a nested list
- Remove drag icon from usage examples.

### CODE REVIEW

File: packages/calcite-components/src/components/list/usage/DragEnabledNested.md [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
   <calcite-list-item open label="Owls">
-    <calcite-list-item
-      label="Great Horned Owl"
-      description="The great horned owl, also known as the tiger owl, or the hoot owl, is a large owl native to the Americas."
-    ></calcite-list-item>
-    <calcite-list-item
-      label="Snow Owl"
-      description="The snowy owl, also known as the polar owl, the white owl and the Arctic owl, is a large, white owl of the true owl family."
-    ></calcite-list-item>
+    <caclite-list>
```
Comment (jcfranco) [inline packages/calcite-components/src/components/list/usage/DragEnabledNested.md:16]: Typo.

**Review by jcfranco — APPROVED** [review]
✨📚✨

Were the existing examples not working or missing the intermediate list for smoother interaction with nested lists?

File: packages/calcite-components/src/components/list/usage/DragEnabledNested.md (hunk 1 above)
Comment (driskull) [reply packages/calcite-components/src/components/list/usage/DragEnabledNested.md:16]: Thanks!

**Comment by driskull** [comment]
> Were the existing examples not working or missing the intermediate list for smoother interaction with nested lists?

Nothing missing, we just want to make the enabling of dragEnabled smoother and promote this HTML.

**Pull request merged**